sdk/purview/purview-datamap-rest/test/public/entityTest.spec.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import { createClient } from "./utils/recordedClient";
+import { createFileFromStream } from "../../src/index";
+import fs from "fs";
+
+
+describe("purview datamap entity test", () => {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Import entity business metadata ", async function () {
+	const client = await createClient(recorder);
+   
+	const response = await client.path("/atlas/v2/entity/businessmetadata/import").post({
+	contentType: "multipart/form-data",
+	body: {
+		file: createFileFromStream(() => fs.createReadStream("test/public/template_2.csv"), "template_2.csv"),
+	}
+	});
+	assert.strictEqual(response.status, "200");
```

Comment:
I notice the import status is failed in recording, is this expected?
```
 "failedImportInfoList": [
          {
            "parentObjectName": "",
            "childObjectName": "",
            "importStatus": "FAILED",
            "remarks": "Line #2: invalid entity-type \u0027hive_database\u0027"
          }
        ]
      }
```

Reply:
Yes. It is expected. The test only wants to ensure the service consumes the multiform data correctly.

Reply:
We do not have much traffic for this API usage. We keep this only to make compatible with Atlas. I think for first beta release, it is not necessary to ensure the import metadata status is successful.